protocol-units/bridge/README.md

```diff
@@ -0,0 +1,120 @@
+    "eth_rpc_connection_protocol": "http",
+    "eth_rpc_connection_hostname": "0.0.0.0",
+    "eth_rpc_connection_port": 8090,
+    "eth_ws_connection_protocol": "ws",
+    "eth_ws_connection_hostname": "0.0.0.0",
+    "eth_ws_connection_port": 8090,
+    "eth_chain_id": 3073,
+    "eth_initiator_contract": "0xb33d9fA868054FD3F8F02ba9Fc04B2a1862Ab855",
+    "eth_counterparty_contract": "0xA6bB0d5eA03f3bEe3510EA3Ec39e85Cd7395Bc37",
+    "eth_weth_contract": "0xB7f8BC63BbcaD18155201308C8f3540b07f84F5e",
+    "signer_private_key": "<ETH_PRIVATE_KEY>",
+    "time_lock_secs": 60,
+    "gas_limit": 10000000000000000,
+    "transaction_send_retries": 10
+  },
+  "movement": {
+    "movement_signer_address": "<MOVEMENT_PRIVATE_KEY>",
+    "movement_native_address": "0xf7c46960ef7aa1b25cd5d48efef064525aa4ccec286fc3b3a7176bbfec09b84c",
+    "mvt_rpc_connection_protocol": "http",
+    "mvt_rpc_connection_hostname": "0.0.0.0",
+    "mvt_rpc_connection_port": 30731,
+    "mvt_faucet_connection_protocol": "http",
+    "mvt_faucet_connection_hostname": "0.0.0.0",
+    "mvt_faucet_connection_port": 30732,
+    "mvt_init_network": "custom"
+  },
+  "testing": {
+    "eth_well_known_account_private_keys": [
+     <0th EL FROM ANVIL>,
+     <1st EL FROM ANVIL>,
+      ...
+    ]
+  }
+}
+
+```
+Once the relayer is up and listening for events, you can initiate a transfer.
+
+## Bridging MOVE from Ethereum to Movement without a UI
+
+First we need to create the `preimage` or secret, for the transfer. 
+
+1. `cast keccak256 <YOUR_SECRET>`
+
+The secret can be any string. When doing this operation through the UI, a unique secret is automatically generate, if using a CLI, you should always use a unique secret for each transfer.
+
+This will generate your `HASH_LOCK` value.
+
+2. Now we call `initiateBridgeTransfer` on `AtomicBridgeInitiatorMOVE` with the cast CLI
+
+```
+cast send <CONTRACT_ADDRESS> "initiateBridgeTransfer(uint256,bytes32,bytes32)" <MOVE_AMOUNT> <RECIPIENT> <HASH_LOCK> --from <YOUR_ADDRESS> --rpc-url <RPC_URL>
+```
+Where 
+- <CONTRACT_ADDRESS>: The deployed contract address.
+<MOVE_AMOUNT>: The amount of MOVE tokens (in wei) to transfer.
+- <RECIPIENT>: The recipient's address, formatted as a bytes32 string (you'll need to convert it to bytes32 beforehand).
+- <HASH_LOCK>: The hash lock for the transfer, also a bytes32 value.
+- <YOUR_ADDRESS>: Your wallet address, which will be the originator of the transaction.
+- <RPC_URL>: The RPC URL of your Ethereum node or test network.
+
+For example:
+```
+cast send 0xYourContractAddress "initiateBridgeTransfer(uint256,bytes32,bytes32)" 1000000000000000000 0x000000000000000000000000000000000000move 0x1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef --from 0xYourWalletAddress --rpc-url http://127.0.0.1:8545
+```
+The relayer will now call functions on the movement atomic bridge contract. You can inspect the logs of the service to see its progress. 
+
+Copy the `BridgeTransferId` in the logs. 
```

Comment:
It's not very clear how you get the bridge transfer Id from the logs. Perhaps you should add which part of the logs to copy.